config.vim

```diff
@@ -231,7 +231,7 @@ set wildmode=list:longest
 ca w!! w !sudo tee "%"
 
 " tab navigation mappings
-map tt :tabnew 
+map tt :tabnew
```

Comment:
this space is meaningfull and needed, the mapping leaves the cursor at the end of the command, with a space in between, so the user can star typing the path if needed